concent_api/api_testing_common.py

```diff
@@ -242,12 +242,6 @@ def parse_command_line(command_line):
     return cluster_url
 
 
-def get_task_id_and_subtask_id(test_id, case_name):
-    task_id = f'task_{case_name}_{test_id}'
```

Comment:
Those are not valid UUID formats. Valid format is ie. `dcb6524f-2f55-4078-94b4-da2d5169e29c`.